include/mbgl/util/position.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
```

Comment:
This file defines a class in the `mbg::style` namespace, so place it in the `mbgl/style` directory.